src/queries/__tests__/displayValue.breaking.test.tsx

```diff
@@ -0,0 +1,129 @@
+import * as React from 'react';
```

Comment:
should we document somewhere what the .breaking tests are? it might not be clear for someone else without the context to figure it out. It might be global documentation or specific documentation for each .breaking test not sure

Reply:
Yes that's a very good idea ! We'd need to document as well the useBreakingChange flag and the way we handle breaking changes in general. I'm thinking of a documentation in codebase that would be linked in the contributing doc wdyt ? 

However I'd rather write this doc in another pr since this one is already pretty huge

Reply:
Great idea!